examples/encrypting.rs

```diff
@@ -18,10 +18,10 @@ async fn main() -> Result<()> {
 async fn encrypt_to_group(sdk: &IronOxide) -> Result<DocumentId> {
     // start-snippet{encryptToGroup}
     let group_id = create_group(sdk).await?;
-    let message = "This is my secret which a whole group should see.";
+    let message = "This is my secret which a whole group should see.".to_string();
     let encrypted_result = sdk
         .document_encrypt(
-            message.as_bytes(),
+            message.into_bytes(),
```

Comment:
I somehow have a slight preference for: 
```rust
let message = "This is my secret which a whole group should see.";
// ...
message.as_bytes().to_vec();
// or
message.as_bytes.to_owned();
```

Reply:
I think I'd prefer to leave this as it is. I agree that in this case with a contrived `&str` it might make more sense using `as_bytes()`, but I think it's just as likely that consumers will have an owned `String` message instead, in which case this is how they'd turn it into bytes.
Let me know if you'd like to discuss this or have me think on it further.